libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Dashboard/DashboardScenarioTigerCharts.tsx

```diff
@@ -2,8 +2,16 @@
 import React from "react";
 import { Dashboard } from "@gooddata/sdk-ui-dashboard";
 import { idRef } from "@gooddata/sdk-model";
-import { Dashboards } from "../../../../../reference_workspace/workspace_objects/goodsales/current_reference_workspace_objects_tiger";
+
+import * as BearMDObjects from "../../../../../reference_workspace/workspace_objects/goodsales/current_reference_workspace_objects_bear";
+import * as TigerMDObjects from "../../../../../reference_workspace/workspace_objects/goodsales/current_reference_workspace_objects_tiger";
+
+type MDObjectsType = typeof TigerMDObjects & typeof BearMDObjects;
+
+export const MDObjects = (
+    process.env.SDK_BACKEND === "TIGER" ? TigerMDObjects : BearMDObjects
+) as MDObjectsType;
```

Comment:
As the previous comment, we don't need to run UI SDK test for both of backend, so we don't need to IF, ELSE here to select the MDObject